scripts/accept-verifier-changes.ps1

```diff
@@ -0,0 +1,23 @@
+# Similar to `dotnet verify accept` but doesn't create new runtime-specific ".verified" files if a common one exists.
```

Comment:
Not clear why do we need this?
Should we have an entry on CONTRIBUTING.md describing when/why to use it?

Reply:
added an entry to contrib docs - it's useful when you have many (intended) changes in `Verify` snapshots